cmd/go-filecoin/init_daemon_test.go

```diff
@@ -63,26 +60,3 @@ func TestDownloadGenesis(t *testing.T) {
 
 	td.ShutdownSuccess()
 }
-
-func TestImportPresealedSectors(t *testing.T) {
```

Comment:
This test causes CI to hang for some reason. It's a pretty poor test of presealed sectors anyway. The functionality is covered in functional tests like `TestBootstrapMineOnce`.